src/v/cluster/BUILD

```diff
@@ -564,8 +564,8 @@ redpanda_cc_library(
         "//src/v/metrics",
         "//src/v/model",
         "//src/v/net",
-        "//src/v/pandaproxy/schema_registry:config",
-        "//src/v/pandaproxy/schema_registry:subject_name_strategy",
+        "//src/v/pandaproxy:config",
+        "//src/v/pandaproxy:subject_name_strategy",
```

Comment:
these are re-rooted so that panda proxy and schema registry can be compiled together as a monolithic library. not ideal, but I encountered various violations of module boundaries and I don't understand pp/sr well enough to fix these efficiently. will rely on ben/oren to help break this down as follow up work.